### ISSUE TITLE
Problem installing with Python 3.4

### ISSUE THREAD

**Issue by dezoito** [deanmalmgren/textract#110]
i'm trying to install with pip in Ubuntu (Trusty 64) with Python 3.4.3, but the installation fails because of PDF Miner:

``` sh
Downloading/unpacking pdfminer==20140328 (from textract)
  Downloading pdfminer-20140328.tar.gz (4.1MB): 4.1MB downloaded
  Running setup.py (path:/tmp/pip_build_root/pdfminer/setup.py) egg_info for package pdfminer
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip_build_root/pdfminer/setup.py", line 3, in <module>
        from pdfminer import __version__
      File "/tmp/pip_build_root/pdfminer/pdfminer/__init__.py", line 5
        print __version__
                        ^
    SyntaxError: Missing parentheses in call to 'print'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip_build_root/pdfminer/setup.py", line 3, in <module>

    from pdfminer import __version__

  File "/tmp/pip_build_root/pdfminer/pdfminer/__init__.py", line 5

    print __version__

                    ^
```

This [blog entry](http://www.tysonmaly.com/uncategorized/installing-textract-for-python-3/) suggests some workarounds - mainly, editing an exception line and installing pdfminer3k, instead of version 20140328, but that would make automated installs cumbersome.

Are there any other suggestions, or plans to update this in a newer release?

**Comment by michaelhays** [comment]
Also having this issue. It makes it very difficult to install textract in pip-dependent environments like Heroku, even with the workarounds.

**Comment by pmlandwehr** [comment]
Alternately, could we patch to [`PDFMiner.six`](https://pypi.python.org/pypi/pdfminer.six/), which should be compatible with all pythons?

**Comment by awwfishsticks** [comment]
Also having this issue

**Comment by pmlandwehr** [comment]
(See [pull 116](https://github.com/deanmalmgren/textract/pull/116), which is currently dying during the CI but tries to address this.)

**Comment by dezoito** [comment]
@ pmlandwehr , 

Thank you for referencing that PR.

I just attempted to install from the py3 branch (using `sudo pip3 install git+https://github.com/jaraco/textract.git@py3`), but it won't compile on my system.

**Comment by sbadia** [comment]
Hi!

Refs: https://github.com/deanmalmgren/textract/issues/102 and https://github.com/aerkalov/ebooklib/issues/104

If that can help, I've a working install steps on Linux using python 3.5:

```shell
apt update
apt dist-upgrade
apt install python3-pip python3-dev libxml2-dev libxslt1-dev antiword unrtf poppler-utils pstotext tesseract-ocr flac ffmpeg lame libmad0 libsox-fmt-mp3 sox
apt install zlib1g-dev
apt install git
pip3 install git+https://github.com/aerkalov/ebooklib.git
pip3 install git+https://github.com/jaraco/textract.git@py3
```

**Comment by alonsopg** [comment]
I am facing the same issue. Any suggestion?

**Comment by basimms** [comment]
I am all ears on this one as well.

**Comment by deanmalmgren** [comment]
I'm delighted to call this issue closed, thanks to the hard work in #104, an excellent contribution in #116, and some final tweaks to get tests to pass in #126. textract `1.5.0` will be released later today which should hopefully resolve this issue going forward.

Please reopen if there was something else causing an issue here.

**Comment by dezoito** [comment]
Thank you so much Dean.

I'm going to run this in a few minutes and see how it goes.
